src/issues/util.ts

```diff
@@ -246,12 +246,14 @@ export async function createGithubPermalink(manager: PullRequestManager, positio
 }
 
 const VARIABLE_PATTERN = /\$\{(.*?)\}/g;
-export async function variableSubstitution(value: string, issueModel: IssueModel, user?: string): Promise<string> {
+export async function variableSubstitution(value: string, issueModel?: IssueModel, defaults?: PullRequestDefaults, user?: string): Promise<string> {
 	return value.replace(VARIABLE_PATTERN, (match: string, variable: string) => {
 		switch (variable) {
-			case 'user': return user ? user : '';
-			case 'issueNumber': return `${issueModel.number}`;
-			case 'issueTitle': return issueModel.title;
+			case 'user': return user ? user : match;

```

Comment:
should we include this list of variables in the `githubIssues.queries` setting description?

Reply:
Added!